libraries/AP_BoardConfig/canbus_driver.cpp

```diff
@@ -21,18 +21,20 @@
 
   #include <AP_UAVCAN/AP_UAVCAN.h>
   #include <AP_ToshibaCAN/AP_ToshibaCAN.h>
+  #include <AP_uLandingCAN/AP_uLandingCAN.h>
```

Comment:
No need for this include actually. The ToshibaCAN could be removed as well. Neither has parameters so they aren't needed here.